Automattic/jetpack
# Boost: Regenerate generic Cloud Critical CSS automatically

Fixes 48-gh-Automattic/boost-hydra

#### Changes proposed in this Pull Request:
- Clear Cloud CSS and request for fresh on theme change.
- Request for fresh Cloud CSS on post_save.
- The request on post_save will only request to refresh relevant providers.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Define constant `JETPACK_BOOST_CLOUD_CSS` as true.
* Change theme and see if Cloud CSS has a refresh.
* Update/Create a post/page/custom_post_type and see if there is update.
* You can monitor logStash for `feature:jetpack_boost_cloud_css_request` to see the requested providers.
